ochap-backend/pom.xml

```diff
@@ -92,7 +92,13 @@
 				<artifactId>spring-security-test</artifactId>
 				<scope>test</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>jakarta.validation</groupId>
+            <artifactId>jakarta.validation-api</artifactId>
+            <version>2.0.2</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
Please use the validation api from javax instead.

```xml
		<dependency>
			<groupId>javax.validation</groupId>
			<artifactId>validation-api</artifactId>
			<version>2.0.1.Final</version>
		</dependency>
```

Reply:
Jakarta is a particular implementation of javaee. The current implementaion of j2ee is controlled by wildfly.